### PR TITLE
Update Compression.md

### PR DESCRIPTION
**Pull request by JakeMessner**

### CODE REVIEW

File: Documentation/SequentialDataStore/Compression.md [hunk 1]
```diff
@@ -4,24 +4,26 @@ uid: sdsCompression
...
 Supported compression schemes
 -----------------------------
 - ``gzip``
 - ``deflate``
+- ``gzip, deflate`` (either)
 
 Request compression (writing data)
 ----------------------------------
-The body content of an HTTP request can be compressed using the [supported compression schemes](#supported-compression-schemes) allowing you to send stream values to the REST API more efficiently.
+The body content of an HTTP request can be compressed using the [supported compression schemes](#supported-compression-schemes) allowing you to send request content to the REST API more efficiently.
```
Comment (osimarkrogers) [inline Documentation/SequentialDataStore/Compression.md]: Do you mean "send request(ed) content" or "request content"?

1.

Comment (JakeMessner) [reply Documentation/SequentialDataStore/Compression.md]: "send request content" as in: send the content or body of an HTTP request.

User's are able to compress the content of their HTTP request, and send that to the REST API more efficiently than they would uncompressed content..

Comment (ScottPatterson) [reply Documentation/SequentialDataStore/Compression.md]: Maybe "HTTP message content" instead of "request content" would be less confusing?

Comment (osimarkrogers) [reply Documentation/SequentialDataStore/Compression.md]: Hyphenated "request content" per convention in previous section.

**Comment by GiselleLeeOSI** [comment]
Looks good to me.

**Comment by GiselleLeeOSI** [comment]
Closing the PR.

**Pull request closed without merging**